scripts/pipeline/request-ciorchestrator.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+GH_API_ROOT="https://api.github.com"
+GH_BRANCH="mult_arch"
+GH_REPOSITORY="sample-getting-started"
+GH_ORG="OpenLiberty"
+CI_TRIGGER="lgsdocker"
+CI_CONFIG_FILE=".ci-orchestrator/sample-liberty-build.yml"
+pipelineName="Liberty Getting Started Build"
+command="make build-app-pipeline"
+
+
+function main() {
+    parse_arguments "$@"
+    request_ciorchestrator
+}
+
+function print_usage() {
+    script_name=`basename ${0}`
+    echo "Usage: ${script_name} [OPTIONS]"
+    echo ""
+    echo "Kick off of CI Orchestrator job"
+    echo ""
+    echo "Options:"
+    echo "   -u, --user       string  IntranetId to use to authenticate to CI Orchestrator"
+    echo "   --password       string  Intranet Password to use to authenticate to CI Orchestrator"
+    echo "   -b, --branch     string  Github Repository branch"
+    echo "   -r, --repository string  GitHub Repository to use"
+    echo "   --org            string  Github Organisation containing repository"
+    echo "   --trigger        string  Name of trigger within CI Orchestrator config file"
+    echo "   --configFile     string  Location of CI Orchestrator config file"
+    echo "   --command        string  Command to execute on remote machine"
+    echo "   -h, --help               Print usage information"
+    echo ""
+}
+
+
+function parse_arguments() {
+    if [[ "$#" == 0 ]]; then
+        print_usage
+        exit 1
+    fi
+
+    # process options
+    while [[ "$1" != "" ]]; do
+        case "$1" in
+        -u | --user)
+            shift
+            USER=$1
+            ;;
+        --password)
+            shift
+            PASSWORD=$1
+            ;;
+        -b | --branch)
+            shift
+            GH_BRANCH=$1
+            ;;
+        -r | --repository)
+            shift
+            GH_REPOSITORY=$1
+            ;;
+        --org)
+            shift
+            GH_ORG=$1
+            ;;
+        --trigger)
+            shift
+            CI_TRIGGER=$1
+            ;;
+        --configFile)
+            shift
+            CI_CONFIG_FILE=$1
+            ;;
+        --command)
+            shift
+            COMMAND=$1
+            ;;                         
+        -h | --help)
+            print_usage
+            exit 1
+            ;;
+        esac
+        shift
+    done
+}
+
+
+function request_ciorchestrator() {
+    pipelineId=OnePipeline_${PIPELINE_RUN_ID}_${RANDOM}
+    cat >ciorchestrator-submit.json <<EOL
+    {
+        "type": "PipelineTriggered",
+        "ecosystemRouting": "prod",
```

Comment:
While testing you likely want this to be dev, otherwise you will need to push the jenkins jobs to early